lib/auth.ts

```diff
@@ -380,15 +380,16 @@ const getAuth = function (
 	 * 	console.log(userActorId);
 	 * });
 	 */
-	async function getUserActorId(): Promise<number> {
-		const { actor } = (await pine.get({
+	async function getUserActorId(): Promise<number | null> {
```

Comment:
It might be easier (require less changes elsewhere) if you make this function error properly rather than returning `null`. The current master atm errors anyway :disappointed: 
```suggestion
	async function getUserActorId(): Promise<number> {
```